appjs/views/edit_project.js

```diff
@@ -170,85 +210,113 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
   },
 
   pollChange: _.debounce(function(){
+    this.alpaca.childrenByPropertyId["tweet_text"].setValue($('textarea#shareText').val());
```

Comment:
Sets the value of the hidden `tweet_text` field in the alpaca form to that of the `#shareText` element on the `publish` tab.